dask/dataframe/tseries/tests/test_resample.py

```diff
@@ -8,6 +9,14 @@
 import dask.dataframe as dd
 
 
+if LooseVersion(pd.__version__) >= '0.18.0':
```

Comment:
I'd rather keep it in the test file, right next to where it's used. It's just a simple helper for running resample with the correct syntax.